<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Amitesh" failures="2" tests="6" name="Test" time="10.56" errors="0" timestamp="2024-03-20T19:25:29 IST">
  <testcase classname="TestCases.Chkbox2Test" name="testcase1" time="2.408"/>
  <testcase classname="TestCases.Chkbox2Test" name="testcase2" time="0.123"/>
  <testcase classname="TestCases.Chkbox2Test" name="testcase3" time="0.087">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.Chkbox2Test.testcase3(Chkbox2Test.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testcase3 -->
  <testcase classname="TestCases.Chkbox2Test2" name="testcase1" time="2.136"/>
  <testcase classname="TestCases.Chkbox2Test2" name="testcase2" time="0.161"/>
  <testcase classname="TestCases.Chkbox2Test2" name="testcase3" time="0.083">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.Chkbox2Test2.testcase3(Chkbox2Test2.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testcase3 -->
</testsuite> <!-- Test -->
